### ISSUE TITLE
CRC hashes next to the patches titles in the web

### ISSUE THREAD

**Issue by davidgfb** [simeonpilgrim/nikon-firmware-tools#97]
Hi. Could CRC hashes be added next to the patches titles for the web, please? Just to verify that the downloads are correct before patching our cameras.

**Comment by simeonpilgrim** [comment]
tl;dr; answer No

Each patch can co exist with other patches, which means the CRC would need to need to be dynamically calculated. 
Which would imply you trust the tool. 

The next thing to know is you are not "downloading" the patch, your computer is opening the file locally, apply the patch locally, and saving to your file system, so the thing you are trying to protect from, be that injection of bad code, is already on your local machine. Or you are trying to prevent man in the middle when saving, albeit, there is no man in the middle as there are not two end, but just one, your web-browser. And if we don't trust the browser, we now have the problem the thing showing you the value you are wanting to check is also the thing you are not trusting..
